var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -442,6 +442,12 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage, CompilerPackage):
             sha256="01ea668489f3f0fb2439060f6c333c4c17ef4c19c4c4e6e6aa4b8ea493e97685",
             when="@14.2.0 target=aarch64:",
         )
+        # Fix from homebrew for macOS 15.4 SDK issue https://gcc.gnu.org/bugzilla/show_bug.cgi?id=119590
```

Comment:
Please remove a separate patch and instead change the range of the previous patch to `04696df09633baf97cdbbdd6e9929b9d472161d3..a495b2dded281beeafec91074e4e82a5a3df8104` (gcc 14.2.0 upstream -> latest commit on the [`gcc-14-2-darwin` branch](https://github.com/iains/gcc-14-branch/commits/gcc-14-2-darwin/)).

No references to Homebrew please, credit goes to Iain Sandoe.